### PR TITLE
Add menu User Activities

### PR DESCRIPTION
**Pull request by sonadztux**
Menu to show all user activities
Screenshot at 2021-12-15 16-20-58

### CODE REVIEW

File: routes.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ func setupRouter(ctrl *controllers.Controller) *gin.Engine {
 
 	// User List
 	r.GET("/user-list", ctrl.GetVisitors)
+	r.GET("/user-activities", ctrl.GetVisitorActivities)
```
Comment (lazuardyk) [inline routes.go]: Ini bikin di comment baru aja kayak ```// User Activities```

File: views/pages/user-activities.tmpl [hunk 2]
```diff
@@ -1,104 +1,100 @@
...
+        "{{ .base_url }}" + "/delete-service/" + id,
+        function (data, status) {
+          console.log("Data: " + data + "\nStatus: " + status);
+          window.location.reload();
+        }
+      );
+    } else {
+      return false
+    }
+  }
```
Comment (lazuardyk) [inline views/pages/user-activities.tmpl]: Fungsi yg ini apus aja nih nan

**Review by lazuardyk — APPROVED** [review]
Work di local gw, sisa minor kode aja. Thanks @sonadztux udah lengkapin fiturnya

File: views/pages/user-activities.tmpl (hunk 2 above)
Comment (sonadztux) [reply views/pages/user-activities.tmpl]: Ok. Gue ga ngeliatan kode yang di bawah haha

Comment (sonadztux) [reply views/pages/user-activities.tmpl]: done

**Comment by sonadztux** [comment]
[UI UPDATED]
The completeness field now uses a progress bar

Screenshot at 2021-12-15 16-37-53

**Pull request merged**